app/controllers/users_controller.rb

```diff
@@ -75,6 +72,19 @@ def destroy
 
   private
 
+  def user_params
+    params.fetch(:user, {}).permit(
+      :first_name, :last_name, :spire, :email, :phone, :active, :reminders_enabled,
+      :change_notifications_enabled, roster_ids: [], membership: [:admin]
+    ).tap do |params|
+      given_roster_ids = params[:roster_ids].map(&:to_i)
+      params[:roster_ids] = (@user.rosters.map(&:id) || []).then do |roster_ids|
+        roster_ids.reject! { |roster_id| !roster_id.in?(given_roster_ids) && @current_user.admin_in?(roster_id) }
+        roster_ids | (given_roster_ids & @user.rosters.map(&:id).to_a)
```

Comment:
This should be

```ruby
roster_ids | (given_roster_ids & whichever_rosters_the_current_user_is_admin_in)
```

Right? As is, I think this prevents adding a user to a roster.

Reply:
The reason I didn't worry about adding to a roster is because creating a user works fine, and there's a "add user from other rosters" input on the manage users page. When editing users, they don't show up as editable unless they're already in the roster you're an admin for.

Reply:
Ok, but I can still click the checkbox. It looked broken to me. In that, I checked off a roster and that did noting.

Reply:
What do you mean when you say you can click it? Meaning you can access the user's edit page when they're not in your roster?

Reply:
No, I'm an admin in two rosters. I get to the user's edit page through roster 1, roster 2 is check-able because I'm an admin in roster 2. I check that box, save, no change to the user.